doc_source/getting-started-eksctl.md

```diff
@@ -63,18 +63,18 @@ After cluster creation is complete, view the AWS CloudFormation stack named `eks
 #### [ Fargate – Linux ]
 
    ```
-   NAME                                                    STATUS   ROLES    AGE     VERSION              INTERNAL-IP       EXTERNAL-IP   OS-IMAGE         KERNEL-VERSION                  CONTAINER-RUNTIME
-   fargate-ip-192-168-141-147.region-code.compute.internal Ready    <none>   8m3s    v1.24.7-eks-7c9bda   192.168.141.147   <none>        Amazon Linux 2   5.4.156-83.273.amzn2.x86_64   containerd://1.3.2
-   fargate-ip-192-168-164-53.region-code.compute.internal  Ready    <none>   7m30s   v1.24.7-eks-7c9bda   192.168.164.53    <none>        Amazon Linux 2   5.4.156-83.273.amzn2.x86_64   containerd://1.3.2
+   NAME                                                    STATUS   ROLES    AGE     VERSION               INTERNAL-IP       EXTERNAL-IP   OS-IMAGE         KERNEL-VERSION                  CONTAINER-RUNTIME
+   fargate-ip-192-168-141-147.region-code.compute.internal Ready    <none>   8m3s    v1.24.6-eks-1558457   192.168.141.147   <none>        Amazon Linux 2   4.14.294-220.533.amzn2.x86_64   containerd://1.6.6
+   fargate-ip-192-168-164-53.region-code.compute.internal  Ready    <none>   7m30s   v1.24.6-eks-1558457   192.168.164.53    <none>        Amazon Linux 2   4.14.294-220.533.amzn2.x86_64   containerd://1.6.6
```

Comment:
Just create latest fargate profile and its version should be `v1.24.6-eks-1558457` and kernel version is `4.14.x` but not `5.4.x`.